.github/workflows/.deployer.yml

```diff
@@ -151,3 +170,22 @@ jobs:
 
             # Completed pod cleanup
             oc delete po --field-selector=status.phase==Succeeded || true
+
+  promote:
+    name: Promote Images
+    if: inputs.promote_images != ''
```

Comment:
I believe we should also check for promote_tags to be not blank

Reply:
Whoops and resolved!